starport/services/chain/build.go

```diff
@@ -169,6 +169,22 @@ func (c *Chain) buildProto(ctx context.Context) error {
 			),
 		)
 	}
+
+	// generate ts client code as well if it is enabled.
+	// NB: Vuex generates JS client code as well but within a vuex store.
+	if conf.Client.Typescript.Path != "" {
+		tsRootPath := filepath.Join(c.app.Path, conf.Client.Typescript.Path, "generated")
```

Comment:
I would remove `generated` as last component to give full control to the user on the path for generation

Reply:
@lubtd are you running `starport serve -r --rebuild-proto-once`? Without the flag, it doesn't generate. This is something we should rethink, but not in this PR, of course.

As for `generated`, this is the way it works for `vuex`.

Reply:
Thanks I was acutally missing the `--rebuild-proto-once` flag

For `generated` yes I realize this is the path for your typescript project and then you have the `generated` folder for automatically generated files